vector/src/main/res/values/strings.xml

```diff
@@ -874,7 +874,7 @@
 
     <string name="encryption_information_verify_device">Verify device</string>
     <string name="encryption_information_verify_device_warning">To verify that this device can be trusted, please contact its owner using some other means (e.g. in person or a phone call) and ask them whether the key they see in their User Settings for this device matches the key below:</string>
-    <string name="encryption_information_verify_device_warning2">If it matches, press the verify button below. \nIf it doesn\'t, then someone else is intercepting this device and you should probably blacklist it.\nIn future this verification process will be more sophisticated.</string>
+    <string name="encryption_information_verify_device_warning2">If it matches, press the verify button below. If it doesn\'t, then someone else is intercepting this device and you should probably blacklist it. In the future this verification process will be more sophisticated.</string>
```

Comment:
Why have you delete all the `\n` ?

Reply:
It looked kind of odd with only a single newline, I figured either two newlines or zero newlines would work better. I can revert it if you don't like the change.

Reply:
ok, let's keep your change for the moment